### DISCUSSION TITLE
Mixing tracks

### DISCUSSION THREAD

**Discussion by atmix** [AzuraCast/AzuraCast#4000] [Q&A]
Hello everyone, I apologize for my bad English, but I will try to explain. Sometimes I put on the air a recording with greetings and wishes, how to overlay this recording over the next track? To play the next track as a background?

**Comment by silvaalex2021** [comment]
Do you mean a Voiceover? I would love that too, but I believe that it is not (and will not be) possible to do this with azuracast, as this goes far beyond what a radio management panel should do.

**Comment by atmix** [comment]
I thought it could be configured here

![45b7511732072b039691e1ce8adccfbd](https://user-images.githubusercontent.com/75258577/114268497-d8ad5f00-9a1a-11eb-98c9-e4c81f6c7a18.png)

**Comment by silvaalex2021** [comment]
I think it’s not possible to get what you want with the options in this dialog.
